### PR TITLE
fix: parameters nuget icon

### PR DESCRIPTION
**Pull request by amirkaws**
> Please provide the issue number

Issue number: #270 

### Changes

> Please provide a summary of what's being changed

This PR adds AWS Icon to the Parameters NuGet Package

### User experience

> Please share what the user experience looks like before and after this change

## Checklist

Please leave checklist items unchecked if they do not apply to your change.

* [x] [Meets tenets criteria](https://awslabs.github.io/aws-lambda-powertools-dotnet/tenets)
* [x] I have performed a self-review of this change
* [x] Changes have been tested
* [x] Changes are documented
* [x] PR title follows [conventional commit semantics](https://github.com/awslabs/aws-lambda-powertools-dotnet/blob/develop/.github/semantic.yml)

<details>
<summary>Is this a breaking change?</summary>

**RFC issue number**:

Checklist:

* [ ] Migration process documented
* [ ] Implement warnings (if it can live side by side)

</details>

## Acknowledgment

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Disclaimer**: We value your time and bandwidth. As such, any pull requests created on non-triaged issues might not be successful.

### CODE REVIEW

File: version.json [hunk 1]
```diff
@@ -6,6 +6,6 @@
     "Tracing": "1.1.0"
   },
   "Utilities": {
-    "Parameters": "0.0.1-preview"
+    "Parameters": "0.0.2-preview"
```
Comment (hjgraca) [inline version.json:9]: Do we need to update the version? There are no code changes. If we push to nuget does it not update if the version is there already?

Comment (sliedig) [reply version.json:9]: I wouldn't say so

Comment (amirkaws) [reply version.json:9]: There is a change in dll metadata at least

Comment (amirkaws) [reply version.json]: Removed the version change, icon will be fixed with next release.

**Review by hjgraca — APPROVED** [review]

**Pull request merged**